### DISCUSSION TITLE
Add support for `:not(:placeholder-shown)`

### DISCUSSION THREAD

**Discussion by ajmnz** [ben-rogerson/twin.macro#569] [Ideas]
Would be really sweet to get this property supported, given that there already are other `:not`-related selectors like `not-checked:`, `not-disabled:`, etc.

Thanks!

**Comment by ben-rogerson** [comment]
Sounds like a great addition, certainly a selector I've had to use before too!

It'll land in the next version of twin - see #570

In the meantime, you can use twins new arbitrary variant feature:

```js
<input tw="text-black focus:[:not(:placeholder-shown)]:text-red-500" placeholder="hey there" />
```

**Reply by ajmnz** [reply]
Awesome, thanks! Left a comment in your pr (#570) 👍🏼
